actions/autosync/auto-sync-entrypoint.sh

```diff
@@ -8,38 +8,38 @@ source /common.sh
 set_git_safe_directory
 
 # Initialize the command variable
-command="trestlebot-autosync \
-        --markdown-path=\"${INPUT_MARKDOWN_PATH}\" \
-        --oscal-model=\"${INPUT_OSCAL_MODEL}\" \
-        --ssp-index-path=\"${INPUT_SSP_INDEX_PATH}\" \
-        --commit-message=\"${INPUT_COMMIT_MESSAGE}\" \
-        --pull-request-title=\"${INPUT_PULL_REQUEST_TITLE}\" \
-        --branch=\"${INPUT_BRANCH}\" \
-        --file-patterns=\"${INPUT_FILE_PATTERN}\" \
-        --committer-name=\"${INPUT_COMMIT_USER_NAME}\" \
-        --committer-email=\"${INPUT_COMMIT_USER_EMAIL}\" \
-        --author-name=\"${INPUT_COMMIT_AUTHOR_NAME}\" \
-        --author-email=\"${INPUT_COMMIT_AUTHOR_EMAIL}\" \
-        --working-dir=\"${INPUT_REPOSITORY}\" \
-        --target-branch=\"${INPUT_TARGET_BRANCH}\" \
-        --skip-items=\"${INPUT_SKIP_ITEMS}\" \
-        --version=\"${INPUT_VERSION}\""
+command="trestlebot autosync \
+        --markdown-dir=\"${INPUT_TRESTLEBOT_MARKDOWN_DIR}\" \
```

Comment:
These inputs might need some testing the `INPUT_ENV_VARIABLE` is generated by GitHub Actions based on the Action inputs. They would most likely not have `TRESTLEBOT` in the name.

Reply:
Thanks, removed the TRESTLEBOT prefix in env variables.